tuna.go

```diff
@@ -1067,7 +1071,8 @@ func measureDelay(ctx context.Context, nodes types.Nodes, concurrentWorkers, num
 				addr := node.Metadata.Ip + ":" + strconv.Itoa(int(node.Metadata.TcpPort))
 				delay, err := tunaUtil.DelayMeasurementContext(ctx, tcp4, addr, timeout, dialContext)
 				if err != nil {
-					if _, ok := err.(net.Error); !ok {
+					var e net.Error
+					if !errors.As(err, &e) {
```

Comment:
Should this be `errors.Is`?

Reply:
> Should this be `errors.Is`?

`erross.As`  try to find if `e` is some kind of `net.Error`